zebra-chain/src/history_tree.rs

```diff
@@ -392,7 +392,7 @@ impl Clone for NonEmptyHistoryTree {
             ),
             InnerHistoryTree::OrchardOnward(_) => InnerHistoryTree::OrchardOnward(
                 Tree::<OrchardOnward>::new_from_cache(
-                    self.network,
+                    &self.network.clone(),
```

Comment:
Nitpick: It may not be necessary to clone here (though cloning an `Arc` is very cheap, so it's not very important either way.)

Reply:
removed